starter/js/atm.js

```diff
@@ -1,43 +1,54 @@
 //Begin with the document ready function
+$(document).ready(function(){
 
+}
```

Comment:
It looks like you've started your document.ready function but you aren't putting any code inside of it. Also, when you first open up your program in the browser, there is an error that says "atm.js:4 Uncaught SyntaxError: missing ) after argument list" which tells me that there is a missing ) somewhere near the top of your code. It should look like this;

```js
$(document).ready(function(){
   // all code goes here
});
```